go.mod

```diff
@@ -39,6 +39,7 @@ require (
 	google.golang.org/genproto v0.0.0-20211208223120-3a66f561d7aa
 	google.golang.org/grpc v1.48.0
 	google.golang.org/protobuf v1.28.1
+	gopkg.in/yaml.v2 v2.4.0
```

Comment:
What caused this to move from indirect to direct?

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/5892